### PR TITLE
Faster idle timeout

### PR DESCRIPTION
**Pull request by vkallerud**
_Features:_
- Idle timeout faster (now 15 secsonds). 
- Idle timeout modal will not appear from home screen ('/welcome'). In stead, it will redirect the user directly back to the splash screen ('/') after 15 seconds.

### CODE REVIEW

File: src/environments/environment.computas.ts [hunk 1]
```diff
@@ -3,6 +3,6 @@ export const environment = {
   WS_ENDPOINT: 'wss://teknisk-museum-backend.azurewebsites.net',
   // TEKNISKBACKEND_ENDPOINT :'https://tekniskback.azurewebsites.net',
   TEKNISKBACKEND_ENDPOINT: 'https://teknisk-museum-backend-singleplayer.azurewebsites.net',
-  inactivityTime: 60 * 1000,
+  inactivityTime: 60 * 250, // 15 seconds
```
Comment (ComputasAlex) [inline src/environments/environment.computas.ts]: Maybe change this to 15 * 1000 to easily see that it is 15 seconds times 1000 milliseconds in a second

File: src/environments/environment.dev.ts [hunk 2]
```diff
@@ -2,6 +2,6 @@ export const environment = {
   production: false,
   WS_ENDPOINT: 'ws://localhost:8000',
   TEKNISKBACKEND_ENDPOINT: 'http://localhost:8000',
-  inactivityTime: 60 * 1000,
+  inactivityTime: 60 * 250, // 15 seconds
```
Comment (ComputasAlex) [inline src/environments/environment.dev.ts]: same here

File: src/environments/environment.ts [hunk 3]
```diff
@@ -7,7 +7,7 @@ export const environment = {
   // WS_ENDPOINT: 'wss://tekniskback-mp.azurewebsites.net',
   WS_ENDPOINT: 'ws://localhost:5000',
   TEKNISKBACKEND_ENDPOINT: 'http://localhost:8000',
-  inactivityTime: 600 * 1000,
+  inactivityTime: 60 * 250, // 15 seconds
```
Comment (ComputasAlex) [inline src/environments/environment.ts]: yes

File: src/environments/environment.tekniskmuseumprod.ts [hunk 4]
```diff
@@ -3,6 +3,6 @@ export const environment = {
   WS_ENDPOINT: 'wss://tekniskmuseumbackendmultiplayer-dev.azurewebsites.net',
   // TEKNISKBACKEND_ENDPOINT :'https://tekniskback.azurewebsites.net',
   TEKNISKBACKEND_ENDPOINT: 'https://tekniskmuseumbackendsingleplayer-dev.azurewebsites.net',
-  inactivityTime: 60 * 1000,
+  inactivityTime: 60 * 150, // 15 seconds
```
Comment (ComputasAlex) [inline src/environments/environment.tekniskmuseumprod.ts]: This is 9 seconds

**Review by ComputasAlex — CHANGES_REQUESTED** [review]
See comments for requested changes. Really liked it while testing though. This should be a focus point during user testing at the museum

File: src/environments/environment.tekniskmuseumprod.ts (hunk 4 above)
Comment (vkallerud) [reply src/environments/environment.tekniskmuseumprod.ts]: Hahaha oops

**Review by ComputasAlex — APPROVED** [review]
Shweeeet. I like

**Pull request merged**